src/test/java/com/wikia/webdriver/testcases/widgettests/AllTagsTests.java

```diff
@@ -41,8 +41,9 @@ public void prepareTest() {
     widgets.add(new VKWidgetPageObject(driver));
     widgets.add(new WeiboWidgetPageObject(driver));
 
-    // Uncomment after 22-sep
+    //TODO: Uncomment after 22-sep
     //widgets.add(new GoogleFormWidgetPageObject(driver));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)